diff -burN PhpDocumentor-1.2.2.1-orig/new_phpdoc.php PhpDocumentor-1.2.2.1/new_phpdoc.php
--- PhpDocumentor-1.2.2.1-orig/new_phpdoc.php	2003-08-31 10:48:50.899229872 +0200
+++ PhpDocumentor-1.2.2.1/new_phpdoc.php	2003-08-31 11:51:36.698741328 +0200
@@ -51,7 +51,7 @@
 /**
 * common file information
 */
-include_once("phpDocumentor/common.inc.php");
+include_once("PhpDocumentor/phpDocumentor/common.inc.php");
 
 // find the .ini directory by parsing phpDocumentor.ini and extracting _phpDocumentor_options[userdir]
 $ini = phpDocumentor_parse_ini_file($_phpDocumentor_install_dir . PATH_DELIMITER . 'phpDocumentor.ini', true);
