--- ./usr/share/pear/PhpDocumentor/phpDocumentor/Converter.inc	2005-09-18 12:49:06.000000000 +0300
+++ /usr/share/pear/PhpDocumentor/phpDocumentor/Converter.inc	2005-09-18 12:51:25.000000000 +0300
@@ -27,7 +27,7 @@
 /**
  * Smarty template files
  */
-include_once("phpDocumentor/Smarty-2.6.0/libs/Smarty.class.php");
+include_once("Smarty/Smarty.class.php");
 /**
  * Base class for all output converters.
  *
--- ./usr/share/pear/PhpDocumentor/phpDocumentor/Converter.inc	2005-09-18 12:53:14.000000000 +0300
+++ /usr/share/pear/PhpDocumentor/phpDocumentor/Converter.inc	2005-09-18 12:57:03.000000000 +0300
@@ -4824,7 +4824,7 @@
         $templ = new Smarty;
         $templ->use_sub_dirs = false;
         $templ->template_dir = realpath($this->smarty_dir . PATH_DELIMITER . 'templates');
-        $templ->compile_dir = realpath($this->smarty_dir . PATH_DELIMITER . 'templates_c');
+        $templ->compile_dir = "/var/tmp"; # FIXME: is this secure?
         $templ->config_dir = realpath($this->smarty_dir . PATH_DELIMITER . 'configs');
         $templ->assign("date",date("r",time()));
         $templ->assign("maintitle",$this->title);
