--- ./usr/share/pear/data/PhpDocumentor/new_phpdoc.php	2005-10-26 17:39:58.000000000 +0300
+++ /usr/share/pear/data/PhpDocumentor/new_phpdoc.php	2005-10-26 17:47:38.000000000 +0300
@@ -51,7 +51,7 @@
 /**
 * common file information
 */
-include_once("phpDocumentor/common.inc.php");
+include_once("PhpDocumentor/phpDocumentor/common.inc.php");
 
 // find the .ini directory by parsing phpDocumentor.ini and extracting _phpDocumentor_options[userdir]
 $ini = phpDocumentor_parse_ini_file($_phpDocumentor_install_dir . PATH_DELIMITER . 'phpDocumentor.ini', true);
